documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -69,30 +108,81 @@ value.converter.apicurio.registry.url=http://apicurio:8080/api
 value.converter.apicurio.registry.global-id=io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy
 ----
 
-Note that Kafka Connect internally always uses the JSON key/value converters for storing configuration and offsets.
+Internally, Kafka Connect always uses JSON key/value converters for storing configuration and offsets.
+
+// Type: procedure
+// Title: Deploying connectors that use Avro in {prodname} containers
+// ModuleID: deploying-connectors-that-use-avro-in-debezium-containers
+== Deploying with {prodname} containers
+
+In your environment, you might want to use a {prodname} pre-built container to deploy a {prodname} connector that uses Avro serializaion. Follow the procedure here to do that. In this procedure, you build a {prodname} image in which {prodname} uses the Avro converter.  
+
+.Prerequisites
 
-== {prodname} Container Images
+* You have the required permissions on a Kafka cluster.
+* You downloaded the {prodname} connector plug-in that you want to deploy with Avro serialization.
 
-Deploy an Apicurio Registry instance (this example uses a non-production in-memory instance):
+.Procedure
 
-[source,subs="+attributes"]
+. Deploy an instance of {registry}. 
++
+ifdef::community[]
+The following example uses a non-production, in-memory, {registry}  instance:
++
+[source,subs="attributes"]
 ----
 docker run -it --rm --name apicurio \
     -p 8080:8080 apicurio/apicurio-registry-mem:{apicurio-version}
 ----
+endif::community[]
+ifdef::product[]
+.. Choose the OpenShift project in which you want to deploy the {prodname} connector. In the following command, `$NAMESPACE` represents your project.  
+.. Deploy the latest published {registry} operator by running the following command:
++
+`curl -sSL https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/install.yaml | sed "s/{NAMESPACE}/$NAMESPACE/g" | oc apply -f -`
++
+This deploys the latest development version of the {registry} operator from the `master` branch. To deploy other versions, specify a different branch or tag, or edit the operator image reference in the file.
```

Comment:
This surely should be a stable released version, not master?

Reply:
I will ask Steve McCarthy to confirm.

Reply:
Adding @smccarthy-ie , to help us straighten this out.

Reply:
Latest stable released version is `0.0.2` (tag name is the same)

Reply:
@gunnarmorling  I removed this instruction. Let me know if you want me to put back an updated command line.